node/node.go

```diff
@@ -279,6 +280,8 @@ func (n *Node) OnStop() {
 			n.Logger.Error("Error closing listener", "listener", l, "err", err)
 		}
 	}
+
+	n.eventBus.Stop()
```

Comment:
so instead of this:
```
	n.BaseService.OnStop()

	n.Logger.Info("Stopping Node")
	// TODO: gracefully disconnect from peers.
	n.sw.Stop()

	for _, l := range n.rpcListeners {
		n.Logger.Info("Closing rpc listener", "listener", l)
		if err := l.Close(); err != nil {
			n.Logger.Error("Error closing listener", "listener", l, "err", err)
		}
	}

	n.eventBus.Stop()
```

you are suggesting 

```
          defer n.BaseService.OnStop()

	  defer n.Logger.Info("Stopping Node")
	// TODO: gracefully disconnect from peers.
	defer n.sw.Stop()

        defer func() {
	for _, l := range n.rpcListeners {
		n.Logger.Info("Closing rpc listener", "listener", l)
		if err := l.Close(); err != nil {
			n.Logger.Error("Error closing listener", "listener", l, "err", err)
		}
	}
        }()

	defer n.eventBus.Stop()
```

or just `eventBus`? why should `eventBus` be stopped with `defer` while others are not?